### ISSUE TITLE
Text writing bug

### ISSUE THREAD

**Issue by MecoCL** [coppolaemilio/dialogic#1232]
Whenever I use the text function, it doesn't show what I write, only one letter from what I wrote, please fix this!

![image](https://user-images.githubusercontent.com/103614257/191506021-9bd373c1-142d-43dd-9428-4f868a2ace57.png)

**Comment by coppolaemilio** [comment]
This doesn't usually happens. What rendering scale are you using on the editor?

**Comment by MecoCL** [comment]
> This doesn't usually happens. What rendering scale are you using on the editor?

As usual, 2D with Stretch, but it was working fine on 1.4.3 version I guess, but after the new updates, this bug became here like that

**Comment by coppolaemilio** [comment]
I mean in the editor itself, not the game. 
From the menu:
Editor -> Editor Settings -> Display Scale

![image](https://user-images.githubusercontent.com/2206700/191546804-2d4bc135-8286-4e81-b03b-017e10c57477.png)

If it is less than 100% it is prone to failure.

**Comment by MecoCL** [comment]
Nah, it's Auto 100%

**Comment by coppolaemilio** [comment]
Did you change the font size or any other setting for the Godot theme?

**Comment by MecoCL** [comment]
Nope, I even tried some older versions and it didn't show this bug

**Comment by MecoCL** [comment]
3.5
+ did you try the text function of Dialogic 1.4.5? maybe you might get this bug as well...

**Comment by coppolaemilio** [comment]
- Made a new project
- installed Dialogic
- Created a new timeline
- Added text events

https://user-images.githubusercontent.com/2206700/191552638-4c1954a1-d6cd-47e8-9b84-b7875a03fec0.mp4


I know there is something weird in your version of dialogic/godot because of how this line is being rendered here from your screenshot:
![image](https://user-images.githubusercontent.com/2206700/191553165-ce8c9dad-bec4-4400-a80c-23de3b78bb09.png)


That line should be centered but on your screenshot is a bit to the left of the center. That means that the scale of the editor is not getting calculated properly which might probably be the same reason why the autosizing of the input box is not working either.

If your rendering scale is 100%, your font sizes are default and everything else is up to date (3.5 stable, 1.4.5) I don't know what's going on :/

**Comment by MecoCL** [comment]
RIP

**Comment by MecoCL** [comment]
I found out the problem, the editor was Auto 75% that's why, I think because my screen is kinda small, sorry for disturbing you

**Comment by coppolaemilio** [comment]
Oh no worries! I don't know if there is an easy fix for making Dialogic rendering properly at 75% scale. Unfortunately I didn't do a good job at making it display good on small screens from the beginning so it is harder to fix now. 
I'll leave the issue open and see if I can do something to fix the input box at least so you are able to write.

**Comment by MecoCL** [comment]
Ok, Thanks!

**Comment by poseisharp** [comment]
I have the same problem, my auto editor scale is at 75%. However, if I change it to 100% it still have the same issue
I am using godot 3.5.1 Dialogic 1.4.5
